payjoin/src/core/receive/v2/mod.rs

```diff
@@ -1307,4 +1307,22 @@ pub mod test {
         assert_ne!(uri.extras.endpoint, EXAMPLE_URL.clone());
         assert_eq!(uri.extras.output_substitution, OutputSubstitution::Disabled);
     }
+
+    #[test]
+    /// Regression/spec test for BIP-78 and issue #946:
```

Comment:
I didn't think we need all the comments maybe just keep thr description of the test itself